docs/static/spec/openapi/logstash-api.yaml

```
@@ -329,39 +1299,245 @@ paths:
         
         NOTE: The health indicators perform root cause analysis of non-green health statuses. 
         This can be computationally expensive when called frequently.
-        
-      operationId: healthStatus
+
+      operationId: healthReport
       tags:
         - health
       parameters:
-        - name: pretty
-          in: query
-          schema:
-            type: boolean
-          description: >
-            If you append `?pretty=true` to the request, the JSON returned will be pretty formatted. Use it for debugging only!
+        - $ref: "#/components/parameters/pretty"
       responses:
         '200':
           description: Indicates a successful call
           content:
             application/json:
               schema:
-                type: object
-                properties:
-                  indicators:
-                    description: Information about the health of Logstash indicators.
-                    type: object
-                  status:
-                    $ref: '#/components/schemas/Statuses'
+                allOf:
+                  - $ref: '#/components/schemas/Common'
+                  - type: object
+                    properties:
+                      symptom:
+                        type: string
+                        example: "3 indicators are healthy"
+                      indicators:
+                        description: Information about the health of Logstash indicators.
+                        type: object
+                        properties:
+                          pipelines:
+                            type: object
+                            properties:
+                              status:
+                                $ref: '#/components/schemas/Status'
+                              symptom:
+                                type: string
+                              indicators:
+                                type: object
+                                additionalProperties:
+                                  description: "pipeline name"
```

Per info you provided in https://github.com/elastic/logstash/pull/17292#discussion_r1993218699, ideally this would be explained more clearly with text like this:
```suggestion
                                  description: "Additional objects are labelled with the pipeline name."
```

---

I made a change here https://github.com/elastic/logstash/pull/17292/files#diff-0034f162c0c271cf5caf39016c6ce7e3c517e66fbdbb1da35bab7e3e110f425bR1973-R1976